homework/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="main.css">
+  <title>Javascript3</title>
+</head>
+
+<body>
+  <div>
+
+    <header>
+      <h1>HYF Repositories</h1>
+      <button>Javascript3</button>
+    </header>
+
+
+
+    <div class="main__input-container">
```

Comment:
However, in small projects, like this homework, I personally wouldn't use it - unless you explicitly want to practice it :) (or if you are a really really big fan of it ;) )

Reply:
> Did you guys see BEM? 😮 Nice! I don't have any experience with it - from my limited knowledge, I'd say that if you have this `main__input-container` _element_, it would need to be inside a `main` _block_, though?

No , we haven't learn it

Reply:
Okay